### PR TITLE
Update to FLEDGE settings content [DON'T MERGE BEFORE REVIEW]

### PR DESCRIPTION
**Pull request by samdutton**
Don't merge before Eng review.

### CODE REVIEW

File: site/en/docs/privacy-sandbox/fledge/index.md [hunk 1]
```diff
@@ -169,23 +169,21 @@ implemented and what's still in progress.
...
+top-level setting in `chrome://settings/privacySandbox`. During initial testing, people can use the
+Privacy Sandbox setting to opt out of FLEDGE. API callers can't access group membership when users
+browse in Incognito mode, and membership is removed when users clear their site data.
 
-Chrome plans to allow users to see and manage the list of interest groups that
-they've been added to, across the sites they've visited.  As with the Privacy
-Sandbox technologies, user settings may evolve with feedback from users,
-regulators, and others.
+`chrome://settings/adPrivacy/sites` lists sites where the user has been added to an interest group.
+Chrome DevTools [shows interest groups](https://developer.chrome.com/blog/fledge-api/#debugging)
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/fledge/index.md:178]:
```suggestion
Chrome DevTools [shows interest groups](/blog/fledge-api/#debugging)
```

**Review by heyawhite — APPROVED** [review]
LGTM. 1 comment re: self-referencing link

**Review by heyawhite — APPROVED** [review]

**Comment by samdutton** [comment]
Updated branch name: see https://github.com/GoogleChrome/developer.chrome.com/pull/5862.

**Comment by Crazydane99** [comment]
Hey stop.all pull all now.she.has.Lorraine knows.she.is.leesa has.the
fake.cards license the court schedules we.used.all.attorneys number s
martell1 Prince William tell Carol she should.probably.go.on vacay.now her
and Dr especially Kathy

**Comment by Crazydane99** [comment]
She has.underground.map as well Alison alesha Tammy Tanya bout.to loose
everything

**Pull request closed without merging**